test/Makefile

```diff
@@ -5,5 +5,21 @@
 # Author: Pawel Pisarczyk
 #
 
+ifdef KERNEL_TEST_GROUPS
+EMPTY :=
+SPACE := $(EMPTY) $(EMPTY)
+COMMA := ,
+KERNEL_TEST_GROUPS := $(subst $(COMMA),$(SPACE),$(KERNEL_TEST_GROUPS))
+
+CFLAGS += -DKERNEL_TEST
+CFLAGS += -I$(PREFIX_H)unity
+
+UNITYLIB := $(PREFIX_A)unity.a
+
+OBJS += $(PREFIX_O)test/test_main.o
+
+include $(addprefix test/, $(addsuffix /Makefile, $(KERNEL_TEST_GROUPS)))
```

Comment:
I see that using `,` as a separator is quite confusing in `Makeflie`. We can leave it as is, but I would welcome some way to compile and run all tests (would `KERNEL_TEST_GROUPS=*` work?)